packages/block-editor/src/hooks/position.scss

```diff
@@ -1,18 +0,0 @@
-.block-editor-hooks__position-selection__select-control {
-	.components-custom-select-control__hint {
-		display: none;
-	}
-}
```

Comment:
These overrides shouldn't  be necessary if we don't set the `__experimentalShowSelectedHint` prop on `CustomSelectControl`